### PR TITLE
Favourite fetch

### PR DESCRIPTION
**Pull request by hugocoding21**

### CODE REVIEW

File: backend/src/controllers/favoriControllers.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
+
 const models = require("../models");
```
Comment (Sax593) [inline backend/src/controllers/favoriControllers.js:3]: No eslint disable

File: backend/src/models/ArtworkManager.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
+
 const AbstractManager = require("./AbstractManager");
```
Comment (Sax593) [inline backend/src/models/ArtworkManager.js:3]: Please no eslint disable

File: backend/src/router.js [hunk 3]
```diff
@@ -41,10 +41,11 @@ router.delete("/authors/:id", verifyToken, authorControllers.destroy);
...
 router.get("/favori", verifyToken, favoriControllers.browse);
-router.get("/favori/:id", verifyToken, favoriControllers.read);
-router.put("/favori/:id", verifyToken, favoriControllers.edit);
-router.post("/favori", verifyToken, favoriControllers.add);
-router.delete("/favori/:id", verifyToken, favoriControllers.destroy);
+router.get("/favori/:id", favoriControllers.read);
+router.put("/favori/:id", favoriControllers.edit);
+router.post("/favori", favoriControllers.add);
+router.delete("/favori/:id", favoriControllers.destroy);
+// router.get("/favorites/:id", favoriControllers.read);
```
Comment (Sax593) [inline backend/src/router.js:49]: death code

File: frontend/src/components/accountcard/News/News.jsx [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
 import React from "react";
+import "./News.scss";
 
 function News() {
   return (
-    <div>
+    <div className="ContainerNews">
       <h2>News</h2>
       {/* Contenu spécifique à la page des news */}
     </div>
```
Comment (Sax593) [inline frontend/src/components/accountcard/News/News.jsx:9]: ????

File: frontend/src/components/accountcard/News/News.scss [hunk 5]
```diff
@@ -1,5 +1,5 @@
 .ContainerNews {
   display: flex;
   flex-direction: column;
-  min-height: 70vh;
+  // min-height: 47vh;
 }
```
Comment (Sax593) [inline frontend/src/components/accountcard/News/News.scss:5]: Death Code

File: frontend/src/page/Galerie/card.jsx [hunk 6]
```diff
@@ -1,16 +1,83 @@
...
 import "react-toastify/dist/ReactToastify.css";
+import { AuthContext } from "../../Context/authContext";
+import { useContext } from "react";
 
+// const customStyles = {
+//   overlay: {
+//     background: "#000",
+//   },
+// };
 function Card({ artwork }) {
```
Comment (Sax593) [inline frontend/src/page/Galerie/card.jsx:18]: Death code

**Review by Sax593 — CHANGES_REQUESTED** [review]

**Review by mcgrathmichael — DISMISSED** [review]

**Pull request merged**